CHANGELOG

```diff
@@ -1,3 +1,11 @@
+* Tue Mar 12 2019 Jeanne Greulich <jeanne.greulich@onyxpoint.com> - 8.0.0-0
```

Comment:
Please merge the three 8.0.0 changelog entries to have a cohesive 'this is what changed from 7.0.1'.  The little change I made doesn't warrant attribution.